os/arch/arm/src/armv7-m/up_schedulesigaction.c

```diff
@@ -186,7 +186,6 @@ void up_schedule_sigaction(struct tcb_s *tcb, sig_deliver_t sigdeliver)
 #else
 				current_regs[REG_PRIMASK] = 1;
 #endif
-				current_regs[REG_XPSR] = ARMV7M_XPSR_T;
```

Comment:
are these changes tested with kernel signal tc's and others?

Reply:
Yes. Kernel tc have been checked.